### PR TITLE
Fixes #12850 - Binds job_invocations with tasks using task groups

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: db/migrate/20151217092555_migrate_to_task_groups.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class MigrateToTaskGroups < ActiveRecord::Migration
+  class FakeJobInvocation < ActiveRecord::Base
+    set_table_name 'job_invocations'
+  end
+
+  def up
+    say 'Migrating from locks to task groups'
+    FakeJobInvocation.where('task_group_id IS NULL').each do |job_invocation|
+      next if job_invocation.task_id.nil?
```
Comment (ohadlevy) [inline db/migrate/20151217092555_migrate_to_task_groups.rb]: include it in the original query?

File: db/migrate/20151217092555_migrate_to_task_groups.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      task_group = JobInvocationTaskGroup.new
+      task_group.tasks << ForemanTasks::Task.find(job_invocation.task_id)
```
Comment (ohadlevy) [inline db/migrate/20151217092555_migrate_to_task_groups.rb]: why not
```ruby
tasg_groups.task_ids << job_invocation.task_id
```

File: db/migrate/20151217092555_migrate_to_task_groups.rb (hunk 1 above)
Comment (adamruzicka) [inline db/migrate/20151217092555_migrate_to_task_groups.rb]: Good point

**Comment by stbenjam** [comment]
@adamruzicka  Thanks much! :+1:  I can view my old invocations now.  Can you squash the commits into 1?

**Comment by stbenjam** [comment]
Adam's gone and want to get the gem's done today.  Squashed in https://github.com/theforeman/foreman_remote_execution/pull/91

**Pull request closed without merging**